html/js/product-multilingual.js

```diff
@@ -375,34 +379,50 @@ function change_image(imagefield, imgid) {
     $("#rotate_left_" + imagefield).click({ imagefield: imagefield, angle: -90 }, rotate_image);
     $("#rotate_right_" + imagefield).click({ imagefield: imagefield, angle: 90 }, rotate_image);
 
-    $('img#crop_' + imagefield).click(function () {
-        $('img#crop_' + imagefield).cropper('clear');
-    });
-
-    $('img#crop_' + imagefield).cropper({
-        "viewMode": 2,
-        "guides": false,
-        "autoCrop": false,
-        "zoomable": true,
-        "zoomOnWheel": false,
-        "zoomOnTouch": false,
-        "toggleDragModeOnDblclick": true,
-        "checkCrossOrigin": false
+    document
+      .getElementById("crop_" + imagefield)
+      .addEventListener("click", () => {
+        croppers[imagefield]?.clear();
+      });
+
+    const imageElement = document.getElementById("crop_" + imagefield);
```

Comment:
The `imageElement` variable could be reused, for example:
```suggestion
    const imageElement = document.getElementById("crop_" + imagefield);
    imageElement.addEventListener("click", () => {
        croppers[imagefield]?.clear();
    });

```